acceptance/bundle/deployment/bind/experiment/script

```diff
@@ -0,0 +1,42 @@
+title "Bind experiment test: "
+
+title "Substitute variables in the template"
+BUNDLE_NAME_SUFFIX=$(uuid)
+export BUNDLE_NAME_SUFFIX
+
+# double slash at the start prevents Windows to apply replacements to the path
```

Comment:
Does this replacement setting comes from some other test? Any chance we can just disable this replacement for this test?